### DISCUSSION TITLE
Existing applications logging to Seq after an https certificate is bound

### DISCUSSION THREAD

**Discussion by Rickos67** [datalust/seq-tickets#1660] [Help]
Will existing applications which log to Seq continue to work on non-http URLs, after an https certificate is bound?

In essence, can both HTTP and HTTPS binding be run in parallel?

**Comment by nblumhardt** [comment]
Hi! Yes, this is fine - as long as Seq's "listen URIs" include both HTTP and HTTPS endpoints, both will work.

If you run into any trouble, please just let us know :+1:

**Reply by Rickos67** [reply]
Thanks Nicholas.
